src/reducers/index.js

```diff
@@ -12,8 +12,15 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return {
+        ...state,
+        count: action.payload + 1
```

Comment:
I'm actually glad you handled it this way now that I see, this is more true to how Redux apps tend to flow.
This is just a pretty bad example App, as Redux in this case is overkill.
You'll see it shine as the apps grow in complexity.

Reply:
In fact, this is how I did it but they added that const initialState which is ... I don't really understand the point of that at all.  In fact, I'm glad you did it this way, because technically this is the one time shallow copies and stuff could be not used and the app function because the real functionality of the app is only concerned with the previous state and that's it - this is great practice to now be able to use this technique for every other redux app since obviously state trees are going to be very complex and we will be doing more logic with them, as will be the case with the todos.